tensorflow/core/kernels/training_ops_test.cc

```diff
@@ -308,4 +308,92 @@ static void BM_PowerSign(int iters, int params) {
 }
 BENCHMARK(BM_PowerSign)->Arg(128 << 10)->Arg(256 << 10);
 
+static void IRpropPlus(int32 n, Graph** init_g, Graph** train_g) {
```

Comment:
I think the python tests are sufficient for these optimizers

Reply:
removed